doc/classes/Input.xml

```diff
@@ -389,6 +480,34 @@
 				[b]Note:[/b] This value can be immediately overwritten by the hardware sensor value on Android and iOS.
 			</description>
 		</method>
+		<method name="set_joy_accelerometer_enabled">
+			<return type="bool" />
+			<param index="0" name="device" type="int" />
+			<param index="1" name="enable" type="bool" />
+			<description>
+				Enables or disables the accelerometer sensor, if available, on the specified joypad. If the operation was successful, returns [code]true[/code], otherwise returns [code]false[/code].
+				[b]Note:[/b] Joypad motion sensors reading is only supported on Windows, Linux and macOS.
+			</description>
+		</method>
+		<method name="set_joy_gyroscope_enabled">
+			<return type="bool" />
+			<param index="0" name="device" type="int" />
+			<param index="1" name="enable" type="bool" />
+			<description>
+				Enables or disables the gyroscope sensor, if available, on the specified joypad. If the operation was successful, returns [code]true[/code], otherwise returns [code]false[/code].
+				[b]Note:[/b] Joypad motion sensors reading is only supported on Windows, Linux and macOS.
+			</description>
+		</method>
+		<method name="set_joy_light">
+			<return type="bool" />
+			<param index="0" name="device" type="int" />
+			<param index="1" name="color" type="Color" />
+			<description>
+				Sets the joypad's LED light, if available, to the specified color. If the operation was successful, returns [code]true[/code], otherwise returns [code]false[/code].
+				[b]Note:[/b] Currently, it's not possible to get the color of the LED light, it's only possible to set it.
```

Comment:
```suggestion
				[b]Note:[/b] There is no way to get the color of the light from a joypad. If you need to use the color for something else, store it in a separate variable.
```
Or something similar

Reply:
Note that technically, this is not flawless in case another process (such as DS4Windows) sets the LED color. In this situation, Godot has no way of seeing the actual LED color displayed on the controller. That said, this is a rare enough use case that it's probably not very important.